lib/Styled/Spacing.jsx

```diff
@@ -9,6 +9,10 @@ import styled from "styled-components";
 //   ${props => props.bottom && `margin-bottom: ${props.bottom * 4}px;`}
 // `;
 export const Spacing = styled.div`
+  // Unsure how to deal with 1px right now
```

Comment:
Maybe we could do it like the way box padding works - there's padding with ratio/multiplier vs normal padding? Alternatively does 0.2 * 5 work if you want it to be scalable? 😆 though I understand it's not very intuitive

Reply:
Yeah wouldn't be too intuitive but more importantly, once/if we change to say, `4`, then you get fractions when you multiply by 0.2 - if it's 1px we just do 1px.. 😬